packages/presentation/lib/screens/movie_details/movie_details_screen.dart

```diff
@@ -116,7 +120,9 @@ class _MovieDetailsScreenState
                   ),
                 ),
                 Positioned(
```

Comment:
@sergei-mikhailovskii-idf  it is possible, but in result I'll give almost full copy with a lot of duplication.

Reply:
-//-

Reply:
Fixed.

Reply:
seems that in this case the solution of extracting into separate widgets was not the best. but at least we tried. in case with list it seems pretty good. thanks!